zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -2943,6 +2981,32 @@ components:
           format: int64
           minimum: 1
 
+    SignalBroadcastRequest:
+      type: object
+      properties:
+        signalName :
+          description: The name of the signal to broadcast.
+          type: string
+        variables:
+          additionalProperties: true
+          description: The signal variables as a JSON object.
+          type: object
+        tenantId:
+          description: The id of the tenant that owns the signal.
+          type: string
+      required:
+        - signalName
+    SignalBroadcastResponse:
+      type: object
+      properties:
+        key:
+          description: The unique ID of the signal that was broadcast.
+          type: integer
+          format: int64
+        tenantId:
+          description: The tenant id of the signal that was broadcast.
```

```suggestion
          description: The tenant ID of the signal that was broadcast.
```

🔧 Consistency of "ID"